druid_setup/single/docker-compose.yml

```diff
@@ -22,6 +22,7 @@ volumes:
   zen_extensions: {}
   zookeeper_data: {}
   zookeeper_datalog: {}
+  zookeeper_logs: {}
```

Comment:
for consistency's sake same changes in clustered? https://github.com/Zenysis/Harmony/tree/master/druid_setup/cluster

Reply:
Hmmm.... good point, but I'm going to skip that. The scope of this PR is really restricted to single server, and I don't have an easy way to test any changes I make to the clustered setup right now.